test/test_call_lua.lua

```diff
@@ -29,10 +30,39 @@ end
 function TestCallLua:printTable(sometable, somestring, table2)
   for k, v in pairs(sometable) do
     print('TestCallLua:printTable ', k, v)
+    if k == 'color' then
```

Comment:
You could actually do simply:
```
assert(sometable['weather'] == 'sunny')
assert(sometable['anumber'] == 10)
...
```
This is a slightly stronger assertion, because then you also check that these keys actually exist.